oshi-core/src/test/java/oshi/hardware/PowerSourceTest.java

```diff
@@ -40,6 +40,10 @@ public void testPowerSource() {
             double epsilon = 1E-6;
             assertTrue(ps.getTimeRemaining() > 0 || Math.abs(ps.getTimeRemaining() - -1) < epsilon
                     || Math.abs(ps.getTimeRemaining() - -2) < epsilon);
+            assertTrue(ps.getHealth() >= 0 && ps.getHealth() <= 1);

```

Comment:
Not sure health is always <=1.  I've seen batteries with current capacity > design capacity.